AirCasting/SessionViews/SingleMeasurementView.swift

```diff
@@ -43,19 +43,19 @@ struct SingleMeasurementView: View {
             }, label: {
                 VStack(spacing: 1) {
                     Text(showStreamName())
-                        .font(Fonts.systemFont1)
```

Comment:
<img width="930" alt="Zrzut ekranu 2022-07-4 o 10 14 57" src="https://user-images.githubusercontent.com/44276070/177112275-9aa03e47-6030-4f5f-aaf9-cb349ea83b8e.png">
IDK to be honest what to do, just pointing it

Reply:
As we discussed, I think we can keep it as it is.